usaspending_api/common/management/commands/resave_instances.py

```diff
@@ -29,10 +29,11 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('model', nargs='+', help='app and model to be re-saved')
+        parser.add_argument(
```

Comment:
OK, I hadn't specifically planned to include these changes in this PR.  Oops.  But!  I do want them anyway - they just add an optional --id argument to `resave_instances`, which is handy for trying it out against just a row or a few rows.